scripts/local/.env

```diff
@@ -0,0 +1,52 @@
+# -----------------------------------------------------------------------------
+# Required Environment Variables for Local setup
+# -----------------------------------------------------------------------------
+#
+# Please locate and paste your string values here to set up 
+# 
+# DO NOT add quotation marks to values
+# -----------------------------------------------------------------------------
+
+# -----------------------------------------------------------------------------
+# IoTHub connection string:
+# {Your IoT Hub} > Shared access policies > Connection string -- primary key
+# -----------------------------------------------------------------------------
+PCS_IOTHUBREACT_ACCESS_CONNSTRING=
+PCS_IOTHUB_CONNSTRING=
```

Comment:
It might be confusing to the user that they need to provide same values for 2 variables. Wondering if we can do this
```
PCS_IOTHUBREACT_ACCESS_CONNSTRING={value}
PCS_IOTHUB_CONNSTRING=PCS_IOTHUBREACT_ACCESS_CONNSTRING
```
Better long term solution would be for us to have env variable names that are same across the microservices.

Reply:
Agreed on the long term solution of eliminating one of these